addon/components/frost-modal-binding.js

```diff
@@ -41,10 +65,48 @@ const FrostModalBinding = Component.extend(PropTypesMixin, {
     return {
       closeOnConfirm: true,
       closeOnOutsideClick: false,
+      confirm: {
+        disabled: false,
+        disabledText: 'Confirm',
+        isVisible: true,
+        text: 'Confirm'
+      },
+      _defaultConfirm: {
+        disabled: false,
+        disabledText: 'Confirm',
+        isVisible: true,
+        text: 'Confirm'
+      },
+      disableConfirmUntilOnConfirmResolves: true,
+      forceDisabledConfirm: false,
       targetOutlet: 'modal'
     }
   },
 
+  // == Computed Properties ===================================================
+
+  @readOnly
+  @computed('confirm', 'forceDisabledConfirm', '_defaultConfirm')
+  /**
+   * Computed our Confirm button's props so that we can alter text/disabled in concord with onConfirm promises
+   * TODO: Update confirm/error/info/warn to use this as well: they currently use this.confirm directly
```

Comment:
Add an issue about this if you haven't already addressed this if.  If you have please remove the comment.

Reply:
I added https://github.com/ciena-frost/ember-frost-modal/issues/127 to describe this.

Reply:
Is it time-intensive to just address these now?

Reply:
My initial impression when I first encountered the linting errors was that it was, but let me look further.

>  Javascript: 52 errors, 0 warnings

It's updating ~50 tests. Most of them should be text changes, so it's probably not so bad.  I'll just update the test names, though, and not the style of expectations.

Reply:
Do you have an example of what you are meaning, as I'm not aware of any linting rules regarding the style of expectations.